### DISCUSSION TITLE
Filtering performance traces by tags

### DISCUSSION THREAD

**Discussion by joelngwt** [getsentry/support#10] [Support]
Hello,

I'm trying to filter performance traces by tags (user id tag) in the `config.traces_sampler`.

I've printed out the `transaction_context`, but somehow there are no tags attached at all. However, when I look at the transaction on the website, the user id tag is there.

```
{
           :trace_id => "1dbefc3674754cde913266979e67f825",
            :span_id => "d6f8500eb8e7fa5c",
     :parent_span_id => nil,
    :start_timestamp => 1653901003.4823456,
          :timestamp => nil,
        :description => nil,
                 :op => "rails.request",
             :status => nil,
               :tags => {},
               :data => {},
               :name => "/tasks",
            :sampled => nil,
     :parent_sampled => nil
}
```
![image](https://user-images.githubusercontent.com/4394614/170956880-5b0420ce-5c8b-47e7-b243-aa7d795c094d.png)

I don't really understand what is happening here. Is it possible to filter by user? I'm doing this because of cost and transaction limits.

**Comment by souredoutlook** [comment]
Hi @joelngwt 

It sounds like the transaction is being started before the user tag is set. Is this a manually instrumented transaction? 

It is possible to filter these out using a custom event processor later in the event lifecycle.
